javascript/javascript2/week3/script.js

```diff
@@ -0,0 +1,145 @@
+// 1. Warmup
+// Log out the text Called after 2.5 seconds 2.5 seconds after the script is loaded.
+
+setTimeout((() => console.log("Called after 2.5 seconds")), 2500);
+
+// Create a function that takes 2 parameters: delay and stringToLog. Calling this function should log out the stringToLog after delay seconds. Call the function you have created with some different arguments.
+
+function delayFunction(delay, stringToLog) {
```

Comment:
Maybe this could have a better name? something like`delayLog`? We already know it is a funcion from the funcion keyword, so there is no need to put it in the name of the funtion, but it is a good practice to try and say what the funcion does in its name. It might also be a good idea to change teh order of parameters so the meassage is first and the delay time is second.